docs/examples/feed-example.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "_id" : "video1",
-    "type" : "video",
+    "type" : "Video",
```

Comment:
Is there a reason you recapitalized these? They need to match the case used in the mongoose schema declaration:

<pre>exports.Listing = FeedItem.discriminator(<b>'listing'</b>, listingSchema);</pre>

We can change the mongoose declaration but we have to choose one or the other.